### PR TITLE
Support Quay.io

### PR DESCRIPTION
**Pull request by phracek**
This commit updates Dockerfile with Quay.io registry

It updates also README.md files with Quay.io and
adds the latest container-common-scripts

### CODE REVIEW

**Comment by phracek** [comment]
The error in the test suite is already reported by https://github.com/sclorg/s2i-php-container/issues/316 and the corresponding pull request is #318

**Comment by phracek** [comment]
[test]

File: README.md [hunk 1]
```diff
@@ -61,7 +67,7 @@ To build a PHP image, choose either the CentOS or RHEL based image:
 
 Alternatively, you can pull the CentOS image from Docker Hub via:
 
-    $ podman pull centos/php-73-centos7
+    $ podman pull centos7/php-73-centos7
```
Comment (fila43) [inline README.md:70]:
```suggestion
    $ podman pull centos/php-73-centos7
    $ podman pull quay.io/centos7/php-73-centos7
```

File: README.md [hunk 2]
```diff
@@ -61,7 +67,7 @@ To build a PHP image, choose either the CentOS or RHEL based image:
 
 Alternatively, you can pull the CentOS image from Docker Hub via:
```
Comment (fila43) [inline README.md:68]:
```suggestion
Alternatively, you can pull the CentOS image from Docker Hub or quay.io via:
```

**Comment by phracek** [comment]
[test][test-openshift]

**Pull request merged**